### ISSUE TITLE
Poor focus indicators in zoomed in debug mode

### ISSUE THREAD

**Issue by kmcnaught** [microsoft/pxt-microbit#6591]
Two contexts where keyboard focus indicators are insufficient:

## Exit debug mode 

At any zoom level, enter debug mode and then tab to the "exit debug mode" button. 

Without focus:
<img width="2556" height="1248" alt="Screenshot showing button with purple background in purple context" src="https://github.com/user-attachments/assets/9bce83f4-7535-491e-b31a-5e24c1f6d138" />

With focus:

<img width="2557" height="1249" alt="Screenshot showing button with slightly darker purple" src="https://github.com/user-attachments/assets/37a2a8ed-bb18-4e1a-94d7-1ca276ce0f0f" />

## Floating sim controls - zoomed in 

With zoom high enough that the simulator 'pops out' from the docked sidebar, the floating sim controls have strange and hard to interpret focus indicators. Also, the tab order is weird so getting to "Exit Debug Mode" is hard. 

[video demo](https://www.loom.com/share/6bec075c0e61472d82ee18c4a64549e9)

**Expected behavior**
I should always be able to see my keyboard focus clearly.
